pkgs/by-name/cp/cplay-ng/package.nix

```diff
@@ -8,19 +8,22 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "cplay-ng";
-  version = "5.2.0";
+  version = "5.3.1";
 
   src = fetchFromGitHub {
     owner = "xi";
     repo = "cplay-ng";
     rev = version;
-    hash = "sha256-M9WpB59AWSaGMnGrO37Fc+7O6pVBc2BDAv/BGlPmo8E=";
+    hash = "sha256-6mphhoLDkGZ2r+elzLlPl3B8fNz3loqrQB8x8276AHI=";
   };
 
   nativeBuildInputs = [
     makeWrapper
+    python3.pkgs.setuptools
```

Comment:
`setuptools` should be in `build-system`, not `nativeBuildInputs`. While `build-system` is merged into `nativeBuildInputs` for now, this could change in the future.